Template.php

```diff
@@ -862,6 +862,10 @@ public function get_identifiers_from_url($url_sent = NULL) {
           // If no URL or website, nothing to worth with.
           return FALSE;
         }
+    } elseif (preg_match('~^' . MAGIC_STRING . '(\S+)$~', $url_sent, $matches)) {
```

Comment:
Should we add a space after `chapter-url` and `url` to ensure that the match doesn't gobble up anything after a URL, such as a comment?  (Not sure whether this could happen in practice, but might not hurt to play it safe.)

Reply:
Space added